tests/setup.py

```diff
@@ -51,7 +51,7 @@ def run_tests(self):
       license="apache2.0",
       packages=find_packages(),
       include_package_data=True,
-      install_requires=["ducktape==0.7.6", "requests==2.20.0"],
+      install_requires=["ducktape==0.7.7", "requests==2.20.0"],
```

Comment:
I wont block on this, but to better facilitate updates to ducktape and not worry about re-pinning on every update, should this instead be: `"ducktape>=0.7.0,<0.8.0"` - As in stay on the 0.7.x line

Reply:
Ah, the eternal debate on version pinning. Arguably doing that is bad because builds are no longer reproducible (which is the problem we had with MarkupSafe). If someone wanted to go back and, e.g., do some testing against a hotfix now, they have to backport this change. If everyone adopts semver and never has bugs, I agree that approach should be safe in theory, but we've also seen it break in a number of ways when trying it.

I don't have strong opinions either way, there are tradeoffs to both. But I would argue none of the approaches are obviously the "right way".